### PR TITLE
Add Quickstart Button to `SidebarPanel` 

### PR DESCRIPTION
**Pull request by ABradham**
This PR adds a button to the `SidebarPanel` component that gives users a quick way to create and test out a Firebase quickstart project in Angular. When pressed, the button:

  1. Prompts the user to select a directory where they want the quickstart to go
  2. Downloads the quickstart into the chosen directory using `git clone` and unix file system shell commands
  3. Opens a new VSCode window to the chosen directory that now contains the quickstart files

The quickstart itself is an Angular Firebase web quickstart ([available here](https://github.com/firebase/quickstart-js/blob/master/firestore/README.md)) that showcases a suite of Firebase tools, including:
  * [Firestore](https://firebase.google.com/docs/firestore/)
  * [Authentication](https://firebase.google.com/docs/auth/)
  * [Functions](https://firebase.google.com/docs/functions/)
  * [Emulators](https://firebase.google.com/docs/emulator-suite)
  * [Storage](https://firebase.google.com/docs/storage/)

The quickstart comes with a number of convenient scripts and a `launch.json` file that allow it to be run with little to no setup once downloaded. 

> **Note:** Because the download script executed when the user presses the `Try a Quickstart` button relies on `git` and unix shell commands, this button is currently unable to used on Windows devices or devices without git installed.

### CODE REVIEW

File: firebase-vscode/src/workflow.ts
Comment (christhompsongoogle) [inline firebase-vscode/src/workflow.ts:130]: According to the style guide, single quotes are preferred:
go/tsjs-style#use-single-quotes

File: firebase-vscode/src/workflow.ts
Comment (christhompsongoogle) [inline firebase-vscode/src/workflow.ts:372]: comment liberally

File: firebase-vscode/src/workflow.ts
Comment (christhompsongoogle) [inline firebase-vscode/src/workflow.ts:21]: Please wait for Christina's approval before merging.

**Review by christhompsongoogle — APPROVED** [review]

File: firebase-vscode/webviews/SidebarApp.tsx
Comment (hsubox76) [inline firebase-vscode/webviews/SidebarApp.tsx:171]: We really need to add a gating mechanism other than "isMonospace" for any new features that haven't been agreed to by the FireSQL team, but I can add it after this PR is merged.

File: firebase-vscode/webviews/SidebarApp.tsx
Comment (hsubox76) [inline firebase-vscode/webviews/SidebarApp.tsx:138]: Is this only used once? Might as well just make it an anonymous function and put it inline?

File: firebase-vscode/src/workflow.ts
Comment (hsubox76) [inline firebase-vscode/src/workflow.ts:383]: Does the output of this show up in the terminal? If not, can you use `pluginLogger.info` to log it? If you don't have time to figure it out, add a todo and I can do it later. At least I think there should be a try/catch on this, that logs the error with `pluginLogger.error`.

Comment (ABradham) [reply firebase-vscode/src/workflow.ts:383]: Gotcha; Wrapped the `execSync` call in a `pluginLogger.info` call added a try/catch block to log any errors in an upcoming commit.

File: firebase-vscode/webviews/SidebarApp.tsx
Comment (ABradham) [reply firebase-vscode/webviews/SidebarApp.tsx:138]: Changed to inline function in an upcoming commit!

File: firebase-vscode/src/workflow.ts
Comment (ABradham) [reply firebase-vscode/src/workflow.ts:384]: 1. All told, it takes about 1-1.5 seconds after you press the button for the files to be downloaded and a new VSCode window to be opened. I'll change the current behavior to include 2 separate `pluginLogger` statements (1 saying `Downloading Quickstart Project` and another for the results of `execSync`)

2. No, the results from `execSync` weren't showing up anywhere before; They only became visible once they were wrapped in `pluginLogger.info` calls.

**Review by hsubox76 — APPROVED** [review]

**Pull request merged**
